src/pages/en/core-concepts/astro-components.mdx

```diff
@@ -107,12 +107,16 @@ const myFavoritePokemon = [/* ... */];
 <p class:list={["add", "dynamic", {classNames: true}]} />
 ```
 
+You're not limited to just Astro components though. You can also include **JavaScript framework components** in your template.
+
+📚 Learn about using [JavaScript framework components](/en/core-concepts/framework-components/) in your Astro project.
+
 ## JSX-like Expressions
 
 You can define local JavaScript variables inside of the frontmatter component script within an Astro component. You can then inject these variables into the component's HTML template using JSX-like expressions!
 
 :::note[dynamic vs reactive]
-Using this approach, you can include **dynamic** values that are calculated in the frontmatter.  But once included, these values are not **reactive** and will never change. Astro components are templates that only run once, at build time.
+Using this approach, you can include **dynamic** values that are calculated in the frontmatter. But once included, these values are not **reactive** and will never change. Astro components are templates that only run during the rendering step.
```

Comment:
```suggestion
Using this approach, you can include **dynamic** values that are calculated in the frontmatter. But once included, these values are not **reactive** and will never change. Astro components are templates that only run once, at build time.
```
Reverting this change, since in the new PR, this part of the text is in a new file. I've copied the suggestion to include in the new Astro Syntax page. This way, all edits in this PR only discuss content staying on the components page.